README.md

```diff
@@ -8,4 +8,17 @@ The predictive performances of the model were evaluated by comparing simulated p
 Schematic illustration of the alveolar and bronchial compartment and subcompartments in the physiologically based biopharmaceutics lung model developed in MoBi.
 
 
-![alt text](https://github.com/Erik-Sjogren/Lung-absorption-model/blob/main/Lung-model.png)
+![alt text](Lung-model.png)
+
+## Code of conduct
+Everyone interacting in the Open Systems Pharmacology community (codebases, issue trackers, chat rooms, mailing lists etc...) is expected to follow the Open Systems Pharmacology [code of conduct](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct).
+
+## Contribution
+We encourage contribution to the Open Systems Pharmacology community. Before getting started please read the [contribution guidelines](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CONTRIBUTING.md#ways-to-contribute). If you are contributing code, please be familiar with the [coding standard](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODING_STANDARDS.md#visual-studio-settings).
+
+## License
+The model code is distributed under the [GPLv2 License](https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/LICENSE).
+
```

Comment:
@Erik-Sjogren  Added all this "standard" content for OSP-READMEs (from https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/MODEL_REPO_README_TEMPLATE.md)